tests/Resources/ti.ui.scrollableview.test.js

```diff
@@ -162,4 +163,76 @@ describe('Titanium.UI.ScrollableView', function () {
 		});
 		win.open();
 	});
+
+	it.ios('preferredIndicatorImage', function (finish) {
+		if (OS_VERSION_MAJOR < 14) {
+			return finish();
+		}
+
+		const view1 = Ti.UI.createView({ id: 'view1', backgroundColor: '#836' });
+		const view2 = Ti.UI.createView({ id: 'view2', backgroundColor: '#246' });
+		const backwardImage = Ti.UI.iOS.systemImage('backward');
+		const forwardImage = Ti.UI.iOS.systemImage('forward');
+		const scrollableView = Ti.UI.createScrollableView({
+			preferredIndicatorImage: backwardImage,
+			views: [ view1, view2 ],
+			showPagingControl: true,
+		});
+
+		win = Ti.UI.createWindow({ extendSafeArea: false });
+		win.addEventListener('postlayout', function listener () {
+			win.removeEventListener('postlayout', listener);
+			try {
+				const preferredBackwardImage = win.toImage();
```

Comment:
Typically this would be something like:
``` js
should(win).matchImage('snapshots/ios/known_good_image.png');
```

But obviously you want to compare against an in-memory `Ti.Blob` already. So I think we'll need to modify the assertion to handle if a Ti.Blob is passed in and not a string.